src/chocolatey/infrastructure/filesystem/IFileSystem.cs

```diff
@@ -196,6 +204,15 @@ public interface IFileSystem
         /// <param name="overwriteExisting">true if the destination file can be overwritten; otherwise, false.</param>
         void CopyFile(string sourceFilePath, string destinationFilePath, bool overwriteExisting);
 
+        /// <summary>
+        ///   Copies an existing file to a new file. Overwriting a file of the same name is allowed.

```

Comment:
Do we need to extend this regarding the ability to make it silent?

Reply:
Same as above.